src/Ledger/LedgerForm/FieldLedgerName/FieldLedgerName.js

```diff
@@ -0,0 +1,45 @@
+import React, { useCallback } from 'react';
+import { Field } from 'react-final-form';
+import { FormattedMessage } from 'react-intl';
+import PropTypes from 'prop-types';
+
+import { stripesConnect } from '@folio/stripes/core';
+import { TextField } from '@folio/stripes/components';
+
+import { ledgersResource } from '../../../common/resources';
+import { validateLedger } from '../validateLedger';
+
+const FieldLedgerName = ({ ledgerId, mutator }) => {
+  const validate = useCallback(value => {
+    return validateLedger(mutator.ledgers, ledgerId, value, 'name');
+  },
+  // eslint-disable-next-line react-hooks/exhaustive-deps
+  [ledgerId]);
```

Comment:
Is useCallback needed here at all?
See https://kentcdodds.com/blog/usememo-and-usecallback

Reply:
so why not? `ledgerId` changes rarely (maybe at first renderings), and then new fn will not be created every render.